mreg_cli/utilities/api.py

```diff
@@ -189,16 +191,19 @@ def auth_and_update_token(username: str, password: str) -> None:
         error(e)
     except requests.exceptions.ConnectionError as err:
         error(err)
+
     if not result.ok:
-        try:
-            res = result.json()
-        except json.JSONDecodeError:
-            res = result.text
-        if result.status_code == 400:
-            if "non_field_errors" in res:
-                raise LoginFailedError("Invalid username/password")
-        else:
-            raise LoginFailedError(res)
+        err = ErrorResponse.to_string(result)
+        # We don't have an error response with a "detail" field
+        if not err:
+            body = result.text
```

Comment:
`requests.Response.text` is a property that is computed every time we access it (no explicit caching involved). Binding the result to a variable ensures we only perform that computation once.